core/src/main/java/com/linecorp/armeria/server/DefaultServerConfig.java

```diff
@@ -143,7 +145,9 @@ final class DefaultServerConfig implements ServerConfig {
             @Nullable Mapping<String, SslContext> sslContexts,
             Http1HeaderNaming http1HeaderNaming,
             DependencyInjector dependencyInjector,
-            List<ShutdownSupport> shutdownSupports) {
+            List<ShutdownSupport> shutdownSupports,
+            boolean shouldLogUncaughtExceptions,
+            long logUncaughtExceptionsIntervalInSeconds) {
```

Comment:
How about merging these two properties into `Duration exceptionReportingInterval`? If it's `Duration.ZERO` or below, it could mean it's disabled. This also allows a user to specify the interval of less than one second.

Reply:
Great idea!